server/src/main/java/io/kroki/server/Server.java

```diff
@@ -89,7 +89,7 @@ static void start(Vertx vertx, JsonObject config, Handler<AsyncResult<HttpServer
     allowedMethods.add(HttpMethod.GET);
     allowedMethods.add(HttpMethod.POST);
     allowedMethods.add(HttpMethod.OPTIONS);
-    router.route().handler(CorsHandler.create("*")
+    router.route().handler(CorsHandler.create(".*")
```

Comment:
Could you please add a comment with the GitHub issue so we don't forget to revert this change once https://github.com/vert-x3/vertx-web/issues/1933 is fixed, merged and released?

Reply:
sure. I updated (ammended) the initial commit to add the comment, please let me know if the phrasing/style is adequate.